recipe/meta.yaml

```diff
@@ -14,6 +14,7 @@ source:
 
 build:
   number: 0
+  skip:  # [py<36]
   script: {{ PYTHON }} -m pip install . --no-deps -vv
   noarch: python
 
```

Comment:
Missing `wheel` package in the host section

Reply:
The package `wheel` was added to the `meta.yaml`